assets/aws/single-ami.json

```diff
@@ -46,7 +46,8 @@
     "tags": {
       "Name": "{{user `ami_name`}}",
       "BuildTimestamp": "{{user `build_timestamp`}}",
-      "BuildType": "production"
+      "BuildType": "production",
+      "teleport.dev/is_public": "true"
```

Comment:
This needs to be in `run_tags` and instead of `tags` for it to apply to the security group.

Reply:
Yeah, it looks like you are right. Unfortunately, the tag will also get inherited by the AMI itself ☹️